rust/scx_bpf_unittests/build.rs

```diff
@@ -86,8 +86,11 @@ fn main() {
                 };
 
                 // unclear where the empty name comes from, filter it out
-                if name != "" {
+                if name != "" && name.chars().all(|c| c.is_alphanumeric() || c == '_') {
+                    println!("Accepted symbol: {:?}", name);
                     tests.push(name);
+                } else {
+                    println!("Skipped suspicious symbol: {:?}", name);
```

Comment:
Thanks for looking into this! I can't reproduce this failure, but does this work for you too? It removes the need for "" for me:

```suggestion
                if symbol.is_definition() {
                    tests.push(name);
                }
```

I don't think we need the debug logging given every symbol is mapped 1->1 to a Rust test.